src/databricks/sql/auth/auth.py

```diff
@@ -79,6 +81,14 @@ def normalize_host_name(hostname: str):
     return f"{maybe_scheme}{hostname}{maybe_trailing_slash}"
 
 
+def get_client_id(hostname: str):
+    return (
+        PYSQL_OAUTH_CLIENT_ID
+        if infer_cloud_from_host(hostname) == CloudType.AWS
+        else PYSQL_OAUTH_CLIENT_ID_AZURE
```

Comment:
what happens if other clouds are used? GCP? should we throw error?

Reply:
The error will be thrown when it tires to get the endpoint configuration if cloud type is neither AWS nor Azure.